osquery/tables/system/windows/shimcache.cpp

```diff
@@ -0,0 +1,220 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <osquery/core.h>
+#include <osquery/filesystem/fileops.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+#include <osquery/tables/system/windows/registry.h>
+#include <osquery/tables/system/windows/userassist.h>
+#include <osquery/utils/conversions/tryto.h>
+
+#include <boost/algorithm/hex.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+
+#include <algorithm>
+#include <string>
+
+const int WIN8 = 256;
```

Comment:
nitpick, for constant global values we use `kCamelCaseNaming` where the initial `k` implies the global-constant (it's an odd pattern I know. ;) ).

Reply:
all variables should be fixed